subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/TestReportIntegrationTest.groovy

```diff
@@ -104,35 +104,36 @@ public class LoggingTest {
         ignoreWhenJupiter()
```

Comment:
Instead of sprinkling deprecation messages in the existing tests here, I would just update the tests to not use the deprecated APIs and add an explicit test that checks for the deprecation messages.

Reply:
SGTM; will do that now.